src/test/java/io/xstefank/wildfly/bot/utils/JsonHandler.java

```diff
@@ -0,0 +1,83 @@
+package io.xstefank.wildfly.bot.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Class responsible for handling and changing properties of the JSON template file.
+ */
+public class JsonHandler {
+
+    private static JsonNode file;
+
+    /**
+     * Constructor for the JsonHandler class.
+     *
+     * @param fileName name of the JSON file located in 'src/test/resources/' folder. The name must contain the .json suffix.
+     * @throws IOException if an error occurred during the JSON file obtaining.
+     */
+    public JsonHandler(String fileName) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        file = objectMapper.readTree(new File("src/test/resources/" + fileName));
+    }
+
+    /**
+     * Sets the action (state) of the GitHub PR.
+     *
+     * @param action the action to set. See {@link Actions}.
+     */
+    public void setAction(Actions action) {
+        ((ObjectNode) file).put("action", action.getValue());
+    }
+
+    /**
+     * Sets the Title of the GitHub PR.
+     *
+     * @param title the title to set.
+     */
+    public void setPullRequestTitle(String title) {
+        ((ObjectNode) file.get("pull_request")).put("title", title);
```

Comment:
I would recommend pulling strings like this into constants so you don't have to change it at several places if GH decides to change their API